src/gui/qgsmapcanvas.cpp

```diff
@@ -1114,8 +1114,9 @@ void QgsMapCanvas::showContextMenu( QgsMapMouseEvent *event )
   };
 
   addCoordinateFormat( tr( "Map CRS — %1" ).arg( mSettings.destinationCrs().userFriendlyIdentifier( QgsCoordinateReferenceSystem::ShortString ) ), mSettings.destinationCrs() );
-  if ( mSettings.destinationCrs() != QgsCoordinateReferenceSystem( QStringLiteral( "EPSG:4326" ) ) )
-    addCoordinateFormat( tr( "WGS84" ), QgsCoordinateReferenceSystem( QStringLiteral( "EPSG:4326" ) ) );
+  QgsCoordinateReferenceSystem wgs84( QStringLiteral( "EPSG:4326" ) );
+  if ( mSettings.destinationCrs() != wgs84 )
+    addCoordinateFormat( wgs84.userFriendlyIdentifier( QgsCoordinateReferenceSystem::ShortString ), wgs84 );
```

Comment:
Can i suggest using the medium string for both these? Ie 
```suggestion
    addCoordinateFormat( wgs84.userFriendlyIdentifier( QgsCoordinateReferenceSystem::MediumString ), wgs84 );
```

That is "EPSG:4326 - WGS 84" (and "EPSG:3857 - WGS 84 / Pseudo-Mercator"). In my training experience a surprising number of users don't have any mental mapping between raw epsg codes and their meaning.

Reply:
Done for all occurrences in the Copy Coordinate context menu.